tests/recipe/test_amplitude_creation.py

```diff
@@ -66,7 +66,7 @@ def test_intensity(self):
             self.particle_list, self.kinematics, self.phsp_sample
         )
         intensity = builder.create_intensity(self.recipe)
-        assert len(intensity.parameters) == 9
+        assert len(intensity.parameters) == 8
```

Comment:
why does this change?

Reply:
Seems that the meson radii are gone.

In the previous commit on the master (`77f50f1e`), `parameters` is:
![image](https://user-images.githubusercontent.com/29308176/91578444-78b1ee80-e94a-11ea-988b-0c32feb2fc7d.png)
while in 29f7dc182c93c0f6f60120d87c10669f59f82559 it's:
![image](https://user-images.githubusercontent.com/29308176/91578881-160d2280-e94b-11ea-922a-ce1cf3201084.png)

Reply:
Ok. We fix that in a separate PR